src/main/seis602/pos/register/Register.java

```diff
@@ -1,5 +1,158 @@
 package main.seis602.pos.register;
 
-public class Register {
+import java.util.ArrayList;
+import java.util.List;
 
+import main.seis602.pos.inventory.Inventory;
+import main.seis602.pos.inventory.Item;
+
+public class Register 
+{
+	private static int identifier = 0;
+	private int registerId;
+	private Cashier cashier;
+	private List<Sale> sales;
+	private Sale activeSale;
+	private double totalSales;
+	private Inventory inventory;
+	
+	public Register(Cashier cashier)
```

Comment:
I think there should be a relationship between the Console app and the Cashier class.
When a new register is initialized, we want to pass in the cashier into the constructor. `Register reg1 = newRegister(cashier)` I took this approach rather than having the console app call `register.SetCashier(cashier)` because I believe setters should typically be used during the middle of application's lifecycle rather than during the initialization of the application (that is what constructors are for imo). I also don't think constructors should be passed parameters that do not correspond to the class variables in its respective class which is why I didn't make the constructor take in `Register(String firstName, String lastName)`. I am open to ideas.